src/token/components/TokenChip.css

```diff
@@ -0,0 +1,25 @@
+.token-chip__button {
```

Comment:
Should we maybe prefix those with `ock`?

`ock_token-chip__button`?

Reply:
yes converting